### ISSUE TITLE
Popper is not correctly placed by `1px` for some viewport widths in `flexbox` container

### ISSUE THREAD

**Issue by piecyk** [FezVrasta/popper.js#713]
### CodePen demo

https://codepen.io/anon/pen/WYOzLr

### Steps to reproduce the problem

1.  resize the codepen window

<img width="663" alt="screen shot 2018-11-16 at 10 20 47" src="https://user-images.githubusercontent.com/82964/48612908-90dd9700-e98a-11e8-9bf5-05105784daa2.png">

<img width="605" alt="screen shot 2018-11-16 at 10 21 08" src="https://user-images.githubusercontent.com/82964/48612904-8fac6a00-e98a-11e8-9223-65eb7827a86b.png">

### What is the expected behavior?

Popper should be placed pixel perfect to reference element 

### What went wrong?

For some viewport widths we got in `computeStyle` `popper.left` very close to integer value, but `Math.flor` in this case will break the offset by 1px

```
popper.left 561.984375
offsets.left 561
```

https://github.com/FezVrasta/popper.js/blob/master/packages/popper/src/modifiers/computeStyle.js#L44 changing it back to `Math.round` looks like fixes the problem

### Any other comments?

Connected with https://github.com/FezVrasta/popper.js/issues/277

**Comment by atomiks** [comment]
Maybe centering needs to use `floor` for left/right, but using shifting like `-start` and `-end` should always `round`. Changing back to `round` will cause the incorrect centering problem... 🤕

**Comment by piecyk** [comment]
Yep, it could be. In case `-start | -end` round looks like better fit. Will try to dig more into it 👍

**Comment by piecyk** [comment]
It make sens that the width of popper don't apply in this scenario as we position to edge of reference element, when cantering it's a different story, IMHO this should resolve this issue 

Thanks for helping with this 👏
